### PR TITLE
kubevirt_vm: add simplified cloud-init definition

### PR DESCRIPTION
**Pull request by machacekondra**

### CODE REVIEW

**Comment by machacekondra** [comment]
@mmazur @pkliczewski @vatsalparekh @adityaramteke

File: lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py [hunk 1]
```diff
@@ -175,17 +181,29 @@
...
+            image: kubevirt/cirros-registry-disk-demo:latest
+
+- name: Start fedora vm with cloud init
+  kubevirt_vm:
+      state: running
+      wait: true
+      name: myvm
+      namespace: vms
+      memory: 1024M
+      cloud_init_nocloud:
```
Comment (pkliczewski) [inline lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py]: Looks ok, I wonder whether we could allow user only provide things like `username`, `password` and other configuration instead if providing base64 content?

Comment (machacekondra) [reply lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py]: Of course, that's what userData is for :)

```yaml
...
cloud_init_nocloud:
        userData: |
          ssh-authorized-keys:
            - ssh-rsa AAAAB3NzaK8L93bWxnyp test@test.com
          password: fedora
          ... etc
```

Comment (machacekondra) [reply lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py]: I can maybe add example of that rather then base64 example

**Comment by mmazur** [comment]
To be clear – this does not prevent a user from doing it the old way, it just adds a new, simpler way, right?

**Comment by machacekondra** [comment]
Currently, yes.

File: lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py (hunk 1 above)
Comment (machacekondra) [reply lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py]: done

**Pull request merged**